frontend/src/components/ui/overflow-scroll.ts

```diff
@@ -7,8 +7,8 @@ import { customElement, property } from "lit/decorators.js";
  * depending on scroll position.
  * @slot
  * @cssPart content
- * @cssproperty --btrix-overflow-scrim-width The width of the scrim. 3rem by default.
- * @cssproperty --btrix-overflow-scroll-scrim-color The color of the scrim. White by default.
+ * @cssproperty --btrix-overflow-scrim-size The size of the scrim. Default set by theme stylesheet.
```

Comment:
For whatever it's worth I landed on "width" to be consistent with properties such as `border-width`, but "size" works too!